addons/link_tracker/models/mail_render_mixin.py

```diff
@@ -34,26 +35,28 @@ def _shorten_links(self, html, link_tracker_vals, blacklist=None, base_url=None)
 
         :return: updated html
         """
+        if not html or is_html_empty(html):
+            return html
         base_url = base_url or self.env['ir.config_parameter'].sudo().get_param('web.base.url')
         short_schema = base_url + '/r/'
-        for match in set(re.findall(tools.HTML_TAG_URL_REGEX, html)):
-            long_url = match[1]
-            # Don't shorten already-shortened links
-            if long_url.startswith(short_schema):
-                continue
+
+        def should_convert_to_tracker(url):
```

Comment:
Maybe we could define the inverse, like ``skip_convert``: shorter and more to the point ?